prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,31 @@
+var languagesArray = [
+  'html',
+  'css',
+  'javascript',
+  'react',
+  'node.js'
+];
+
+console.log(languagesArray);
+console.log(languagesArray.length);
+
+languagesArray.push('C#', 'php');
+console.log(languagesArray);
+
+languagesArray.pop();
+console.log(languagesArray);
+
+languagesArray.unshift('C++');
+console.log(languagesArray);
+
+languagesArray.shift();
+console.log(languagesArray);
+
+var thirdElement = languagesArray[2];
+console.log(thirdElement);
+
+var arrayLength = languagesArray.length;
+console.log(arrayLength);
+
+var lastItem = languagesArray[5];
```

Comment:
Here you would use your `arrayLength` variable to calculate the last index, instead of hard-coding it to `5`.

```js
var lastItem = languagesArray[arrayLength - 1];
```